test/e2e/medusa_test.go

```diff
@@ -134,6 +136,19 @@ func checkMedusaContainersExist(t *testing.T, ctx context.Context, namespace str
 	require.True(found, fmt.Sprintf("%s doesn't have medusa container", dc1.Name))
 }
 
+func checkMedusaStandalonePodExists(t *testing.T, ctx context.Context, namespace string, dcKey framework.ClusterKey, f *framework.E2eFramework, kc *api.K8ssandraCluster) {
```

Comment:
Doesn't `ClusterKey` contain `namespace`? Do we need to pass them separately?

Reply:
Also, can we put this in `pkg/test`? `suite_test.go` is too long and doesn't have much coherence.

Reply:
This isn't even in `suite_test.go`. Ignore me.